### DISCUSSION TITLE
Is there any Discord or Slack Link?

### DISCUSSION THREAD

**Discussion by theanantchoubey** [codex-team/editor.js#2437] [Q&A] [answered]
I want to work on issues, but I am finding some difficulties, setting up the project locally.
I searched everywhere but haven't found Discord Link or any guidelines on how to set up an editor.js project locally.
Please let me know if there are any.

**Comment by alinceDev** [comment, marked as answer]
You have a guide here :  
https://editorjs.io/core-development

**Reply by theanantchoubey** [reply]
Got it, thanks!
